packages/flutter_tools/lib/executable.dart

```diff
@@ -145,6 +145,13 @@ List<FlutterCommand> generateCommands({
     artifacts: globals.artifacts!,
     // new ProjectValidators should be added here for the --suggestions to run
     allProjectValidators: <ProjectValidator>[GeneralInfoProjectValidator()],
+    machineValidators: <ProjectValidator>[
```

Comment:
What we can do is having 2 types of `ProjectValidator`, make a new abstract class that inherits from `ProjectValidator` called `MachineProjectValidator` move all of the validators under `allProjectValidators` and on `validate_project.dart` run and output only the ones needed